webapp/src/components/data_selectors/jira_epic_selector/jira_epic_selector.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Theme} from 'mattermost-redux/types/preferences';
+
+import {isEpicNameField, isEpicIssueType} from 'utils/jira_issue_metadata';
+import {IssueMetadata, ReactSelectOption, JiraIssue} from 'types/model';
+import {Props as ReactSelectSettingProps} from 'components/react_select_setting';
+
+import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+
+const searchDefaults = 'ORDER BY updated DESC';
+
+type Props = BackendSelectorProps & {
+    searchIssues: (params: object) => Promise<{data: JiraIssue[]}>;
```

Comment:
Not specially fan of "generic" types like object. Any chance it can be set to a more specific type?